### ISSUE TITLE
Demo site is broken in Firefox

### ISSUE THREAD

**Issue by filipdanic** [prateekbh/preact-material-components#1274]
I opened https://material.preactjs.com/ in Firefox and… it’s kind of broken. Clicking on the home drawer doesn’t do anything and there is no error in the console.

Opening a link such as https://material.preactjs.com/component/button directly will result in 1/4 of the page having the actual content, and 3/4 of it is blank.

**Comment by filipdanic** [comment]
I should note that I came here via `preact-cli` which I used to bootstrap a preact starter with material ui. The drawer component that came with that template doesn’t exhibit the same bug.

**Comment by cromefire** [comment]
This is known, I found this out will be fixed with the next version, it seems like it's a bug with mdc

**Comment by cromefire** [comment]
> Opening a link such as https://material.preactjs.com/component/button directly will result in 1/4 of the page having the actual content, and 3/4 of it is blank.

I cannot confirm that

**Comment by filipdanic** [comment]
<img width="1920" alt="screen shot 2019-01-30 at 2 44 10 pm" src="https://user-images.githubusercontent.com/7009755/51985188-96348300-249d-11e9-9e67-1dd98151e72e.png">

This is what it looks like on my end.

**Comment by cromefire** [comment]
Oh you mean that, yes that's also known, but I don't know if that is fixed.

**Comment by cromefire** [comment]
But it's a thing that is rather post `2.0`, because it is fixed by the newer components. Until then I would recommend to look at the golden images:
https://github.com/prateekbh/preact-material-components/tree/v1.5.5/docs/tests/golden/wide

Or the material docs:
https://material.io/develop/web/

Sorry

**Comment by prateekbh** [comment]
hah nice find! that's an easy fix.. i'll get it right in

**Comment by cromefire** [comment]
should be fixed

**Comment by prateekbh** [comment]
how?

**Comment by cromefire** [comment]
Oh no sorry I thought that problem did also affect the desktop and you released the docs...

**Comment by prateekbh** [comment]
nah it didnt re-deployed

**Comment by cswl** [comment]
Is there any update? Its still broken in Firefox

**Comment by cromefire** [comment]
Once 2.0 is published it will be fixed

**Comment by DenysVuika** [comment]
I can confirm the problem is still there

**Comment by cromefire** [comment]
Yes bei assured I know that I'm using Firefox myself, this will be fixed with `2.0`
